tools/releaseBuild/Images/GenericLinuxFiles/PowerShellPackage.ps1

```diff
@@ -37,6 +37,13 @@ try {
         "AppImage" { Start-PSPackage -Type AppImage @releaseTagParam }
         "tar"      { Start-PSPackage -Type tar @releaseTagParam }
     }
+
+    if ($ExtraPackage -contains "tar-arm") {
+        ## Build 'linux-arm' and create 'tar.gz' package for it.
+        ## Note that 'linux-arm' can only be built on Ubuntu environment.
+        Start-PSBuild -Runtime linux-arm -PSModuleRestore
```

Comment:
Don't we need to build with at least the release tag?

Reply:
Good catch!